iOSLearningProject/iOSLearningProject/LoginController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  LoginController.swift
+//  UIOne
+//
+//  Created by Srushti Sarju Engineer on 19/02/25.
+//
+
+import UIKit
+
+final class LoginController: UIViewController {
+    @IBOutlet weak var signUpBlue: UILabel!
+    @IBOutlet weak var googleLogo: UIImageView!
+    @IBOutlet weak var googleLoginButton: UIButton!
+    @IBOutlet weak var txtFieldEmail: UITextField!
+    @IBOutlet weak var txtFieldPassword: UITextField!
+    
+    @IBAction func btnLoginClick(_ sender: Any) {
+        print("Login button clicked from login page")
+    }
+    
+    @IBAction func btnLoginFacebook(_ sender: Any) {
+        print("Login using Facebook clicked")
+    }
+    
+    @IBAction func btnLoginGoogle(_ sender: Any) {
+        print("Login using Google clicked")
+    }
+    
+    
+    @IBOutlet weak var btnLogin: UIButton!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        if let signUpBlueText = signUpBlue.text {
+            var attributedString = AttributedString(signUpBlueText)
+            if let rangeOfSignUp = attributedString.range(of: "Sign Up") {
```

Comment:
The string should not be here in plain text.
And the UI setup should be grouped neatly in separate functions